x-pack/plugins/infra/public/components/new_badge.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiBadge } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+export const NewBadge = () => (
+  <EuiBadge color="accent">
+    {i18n.translate('xpack.infra.newBadgeLabel', {
+      defaultMessage: 'NEW',
```

Comment:
Do we really need to Uppercase it?

Reply:
I am not sure that I am the right person to decide that - I saw the profiling badge and the nav menu badge are uppercase so I tried to keep it consistent then I checked that in APM it's not 😕 So IMO if we have the same look everywhere is better but which one is better from the UX perspective (uppercase or lowercase) and how much we want to put the attention there I don't know. I would keep it like that here but we can also open a discussion with the design team and add an issue to align the labels everywhere, wdyt?